superduperdb/server/app.py

```diff
@@ -19,9 +19,9 @@
 
 
 class ExceptionHandlerMiddleware(BaseHTTPMiddleware):
-    async def dispatch(self, request: Request, call_next):
+    def dispatch(self, request: Request, call_next):
         try:
-            return await call_next(request)
```

Comment:
@us good catch, since call_next could be a blocking call (synchronous), async won't be helpful!